### DISCUSSION TITLE
[feature request] LeaderBoard

### DISCUSSION THREAD

**Discussion by Dracathio** [Zrips/CMI#1682] [Feature Requests]
Alright so with leaderhead developer RobiRami no longert doing shit at all... and 1.14 is now here and his plugin is still 1.12.2 and he's now been gone for 2 weeks and he said he'd have something out in feburary... it's now may and well 1.14 is here now and no update... hasn't even given the plugin to those who have said they're willing to work on it and update it etc etc.

I wanted to know if you'd be able to have CMI get a leaderboard/head/sign concept put together that is similar to LeaderHeads.

Where it could connect with all vanilla stats, mcmmo, towny, etc.  I mean idk if you already have something like that or not; but it would be nice if that was possible and more so if it could end up hooking into the hologram concept with a spinning/static head.

Many people use the LeaderHeads plugin and with it just not updating there are people who stayed in 1.12.2 because of it; if you could add this to CMI or even better make your own LeaderHead plugin it would be very much appreciated by the community.

**Comment by mrfloris** [comment]
This is a highly requested feature, to have a single plugin that can handle showing a variety of 'stats' from say top 10 popular plugins with either a sign, a head, a player-npc. While technically I think each plugin could do this, having a single plugin that supports those would mean there's some consistency. 

Personally I feel this is out of the realm of the core reason what cmi is about, so if added, please limit it to a module.yml - so we can globally turn it on/off if we use this or not.
